admin/templates/preprint_providers/update_preprint_provider_form.html

```diff
@@ -1,3 +1,72 @@
+    <head>
+        <script>
+            var TINYMCE_BASE_CONFIG = {
+                nowrap: true,
+                branding: false,
+                entity_encoding: 'raw',
+                link_context_toolbar: true,
+            };
+
+            var DESCRIPTION_CONFIG = $.extend({
+                selector:'textarea[name=description]',
+                valid_elements: '*[*]',
+                plugins: 'link preview',
+                toolbar: 'bold italic underline | undo redo | link | preview',
+                forced_root_block: ''
+            }, TINYMCE_BASE_CONFIG);
+
+            var ADVISORY_BOARD_CONFIG = $.extend({
+                selector:'textarea[name=advisory_board]',
+                content_css: 'https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css',
+                valid_elements: '*[*]',
+                plugins: 'link lists preview',
+                toolbar: 'bold italic underline | undo redo | link | bullist | preview'
+            }, TINYMCE_BASE_CONFIG);
+
+            var FOOTER_LINKS_CONFIG = $.extend({
+                selector:'textarea[name=footer_links]',
+                content_css: [
+                    'https://netdna.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css',
+                ],
+                valid_elements : "@[id|class|style|title|dir<ltr?rtl|lang|xml::lang|onclick|ondblclick|"
```

Comment:
Not for this release, but I'm probably going to soon update the other two to me *much* more restrictive.